QUEST_LV_0100.tsv

```diff
@@ -4735,10 +4735,10 @@ QUEST_LV_0100_20150317_004734	You have defeated Rexipher! Move to the burian cha
 QUEST_LV_0100_20150317_004735	Listen to the advices of Zachariel
 QUEST_LV_0100_20150317_004736	Listen to the interpretations of the revelation from the will of the greatest king of humans, Zachariel.
 QUEST_LV_0100_20150317_004737	Obtain the Burning Stone from the protectors of the Mausoleum
-QUEST_LV_0100_20150317_004738	To purify the polluted magic energy, you should light up the stone lamp with the Burning Stone. Defeat the protector of the Mausoleum first to get the Burning Stone.
+QUEST_LV_0100_20150317_004738	To purify the polluted magic energy, you should light up the stone lamp with the Burning Stone. Defeat Royal Mausoleum Guardian first to get the Burning Stone.
```

Comment:
If there is only one guardian, then the last sentence should be "Defeat the Royal Mausoleum Guardian", otherwise "Defeat **a** Royal Mausoleum Guardian"